bats/fb-katello-content.bats

```diff
@@ -122,11 +122,13 @@ setup() {
 }
 
 @test "export content view version" {
+  tSkipIfPulp339
+
   hammer content-export complete version --organization="${ORGANIZATION}" \
     --content-view="${CONTENT_VIEW}" --version="1.0"
   export_version_id=$(hammer --output csv --no-headers content-view version show --version="1.0" --content-view="${CONTENT_VIEW}" --organization="${ORGANIZATION}" \
     --fields=id)
-  actual_size=$(du -k "$(hammer --output csv --no-headers content-export list --content-view-version-id=$export_version_id --fields="path")"/*.gz | cut -f 1)
+  actual_size=$(du -k "$(hammer --output csv --no-headers content-export list --content-view-version-id=$export_version_id --fields="path")"/*.tar | cut -f 1)
 
  [ $actual_size -ge 40 ]
```

Comment:
Not for this PR, but can we actually list the contents of the tar file and check that? It would also validate the resulting file(s) are actually valid tar files.

As for the rename: don't we need to keep the gzip bits for older Pulp versions?

Reply:
Based on evgeni's suggestion, and verifying with dalley that it is so, I've switched it to do ".tar*' to encapsulate gz without needing to resort to adding complexity with if/else logic